### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by abayomi185** [abayomi185/simple-binance-pump-dump-bot#47] [Announcements]
I just discovered how to set up discussions! I'm happy to answer questions about getting set up.

This should also help reduce the clutter of questions and answers in the issues section.

**Comment by UnknownDemiGod** [comment]
![image](https://user-images.githubusercontent.com/86069035/122430993-f6072800-cf8b-11eb-9e48-b318b015e2c8.png)
Hi, 
Thanks for making a great bot. I was trying to setup up telegram-register. I tried to authenticate my phone number but it doesn't seem to verify me. I don't have 2nd step verification either. Cannot get the bot to verify me. Any ideas what's up. Here is the page for auth. https://core.telegram.org/method/auth.sendCode

**Reply by abayomi185** [reply]
Hey, you should be good to go. There should also be confirmation on Telegram that you're logged in.

I haven't fully implemented the UI for completing the registration process.

**Reply by UnknownDemiGod** [reply]
Sorry if that's not the issue then I'm not sure why I'm getting access denied from node js. I have increased the history count so  the bot can track old messages to match the syntax but it throws this
![image](https://user-images.githubusercontent.com/86069035/122434907-6cf1f000-cf8f-11eb-96f3-cc5f8588e84f.png)

**Reply by abayomi185** [reply]
Have you filled in your API details and config for Kucoin in the #kucoin folder?

**Reply by UnknownDemiGod** [reply]
Yep

**Reply by abayomi185** [reply]
Okay. Please run the bot again with ```node  --trace-warnings bot.js``` and share the error

**Reply by UnknownDemiGod** [reply]
![image](https://user-images.githubusercontent.com/86069035/122439240-62395a00-cf93-11eb-913a-2a710408f056.png)

**Reply by abayomi185** [reply]
It's seems that there's an issue with your Kucoin authentication (difficult to tell though). Two things to check; the name of your Kucoin secrets file (removing template-) and removing the string quotes for your secrets in the secrets file.

**Reply by UnknownDemiGod** [reply]
I don't think it's API keys because the bot can read my balance, but I think I got API banned, so I cannot even try manual buy for now. Will  update in a bit.

**Reply by UnknownDemiGod** [reply]
Don't worry, API ban wasn't the bot doing, I was testing something.

**Reply by abayomi185** [reply]
Ahh, Cool.

**Reply by UnknownDemiGod** [reply]
This is something new. 
![image](https://user-images.githubusercontent.com/86069035/122446997-2e623280-cf9b-11eb-878f-9f0671c025b3.png)
As you can probably see I have selected the scrapper but it doesn't seem to be scrapping. Is there any sort of indication that it is scrapping?
I did increase the history_count to 12 so I can test this.
Btw i can now buy and sell without scrapper option so there is no issue with that

**Reply by UnknownDemiGod** [reply]
Verified my changes. I haven't updated/edited any of the scrapper files.

**Reply by abayomi185** [reply]
Cool. I seem to have the same issue too. The messages have been retrieved but not parsed. I'm doing some testing to get it resolved in a few minutes.

**Reply by abayomi185** [reply]
All sorted. There were false positive while parsing the coin due to this: ```suffix_to_split: "-" # "-" in DOGE-USDT => DOGE``` for the scraper config in conf.yaml file. The scraper does not work if the coin name is invalid. Please leave it blank for the time being.

```suffix_to_split: "" # "-" in DOGE-USDT => DOGE```

**Reply by UnknownDemiGod** [reply]
I can see something is improved but I think it's still parsing wrong text on my side.
![image](https://user-images.githubusercontent.com/86069035/122452877-a3386b00-cfa1-11eb-9574-e3d147a1bc46.png)

**Reply by abayomi185** [reply]
I just pushed an update. Only two files changes, telegram.js and bot-kucoin.js. You can put the new suffix condition as "-USDT" for the additional check.

**Comment by PieroMF2010** [comment]
Hi. I was testing the Telegram Scraper, but I am stucked in the phone verification. Any suggestions?
![image](https://user-images.githubusercontent.com/86988384/124522257-3b16c100-ddb8-11eb-9c92-b04990e10038.png)
Thank you so much for your help

**Reply by abayomi185** [reply]
You should be able to proceed after pressing enter. A code would be sent to your telegram which you will need to input at the next prompt, after your phone number.

You can ignore the error messages.

**Reply by PieroMF2010** [reply]
Many thanks for your quick reply. I am not receiving any code and I needed to stop the code. Is there any special configuration in the telegram account? Maybe you can give me a hint about this. Thank you in advance.
![image](https://user-images.githubusercontent.com/86988384/124523732-e4ac8100-ddbd-11eb-8dff-d9336b89afb0.png)

**Comment by liberwicca** [comment]
![image](https://user-images.githubusercontent.com/30032968/231689791-daa22739-6b60-44a6-91ff-abbed56a60a9.png)
Does anyone know how to fix this error, please help me

**Reply by abayomi185** [reply]
Hey, I’ve lost interest in this project but it seems like the quotes are encoded differently on your system.
